src/functions/questionnaire/update-questionnaire/handler.ts

```diff
@@ -0,0 +1,49 @@
+import { ValidatedEventAPIGatewayProxyEvent } from "@libs/api-gateway";
```

Comment:
Perhaps use type of here ?
```suggestion
import type { ValidatedEventAPIGatewayProxyEvent } from "@libs/api-gateway";
```